src/cljs/rems/applications.cljs

```diff
@@ -3,20 +3,23 @@
             [rems.application-list :as application-list]
             [rems.spinner :as spinner]
             [rems.text :refer [localize-state localize-time text]]
-            [rems.util :refer [fetch]]))
+            [rems.util :refer [fetch]]
+            [rems.roles :as roles]))
 
 (rf/reg-event-fx
  ::enter-page
  (fn [{:keys [db]} _]
    {:db (dissoc db ::my-applications)
```

Comment:
Current logic dissocs :my-applications, when fetch starts. Should same be done to :all-applications?

I think that either all applications should follow the same logic, or the discrepancy should be documented.